agent/util-scripts/pbench-move-results

```diff
@@ -284,7 +298,7 @@ for dir in `/bin/ls -ort -d */ | awk '{print $8}' | grep -v "^tools-" | grep -v
     # set the state of the result appropriately so that it will be processed
     # by the server scripts.
     ssh -i $pbench_bin/id_rsa $ssh_opts $results_repo \
-	/opt/pbench-server/bin/pbench-server-set-result-state $results_full_path $tarball
+	/opt/pbench-server/bin/pbench-server-set-result-state $hostnames_full_path $results_full_path/$tarball
```

Comment:
This is unnecessary in the new version: again, the only thing `pbench-move-results` should do is copy the tarball to the specified directory on the server.